spec/units-of-measure.md

```diff
@@ -1,138 +1,133 @@
 
 ```fsgrammar
 measure-literal-atom :=
-    long-ident                  -- named measure e.g. kg
-    ( measure-literal-simp )    -- parenthesized measure, such as (N m)
+    long-ident                                  -- named measure e.g. kg
+    ( measure-literal-simp )                    -- parenthesized measure, such as (N m)
 
 measure-literal-power :=
     measure-literal-atom
-    measure-literal-atom ^ int32 -- power of measure, such as m^3
+    measure-literal-atom ^ int32                -- power of measure, such as m^3
 
 measure-literal-seq :=
     measure-literal-power
     measure-literal-power measure-literal-seq
 
 measure-literal-simp :=
-    measure-literal-seq         -- implicit product, such as m s^- 2
+    measure-literal-seq                         -- implicit product, such as m s^- 2
     measure-literal-simp * measure-literal-simp -- product, such as m * s^3
     measure-literal-simp / measure-literal-simp -- quotient, such as m/s^2
-    / measure-literal-simp      -- reciprocal, such as /s
-    1                           -- dimensionless
+    / measure-literal-simp                      -- reciprocal, such as /s
+    1                                           -- dimensionless
 
 measure-literal :=
-    _ -- anonymous measure
-    measure-literal-simp        -- simple measure, such as N m
+    _                                           -- anonymous measure
+    measure-literal-simp                        -- simple measure, such as N m
 
 const :=
     ...
-    sbyte < measure-literal >   -- 8 - bit integer constant
-    int16 < measure-literal >   -- 16 - bit integer constant
-    int32 < measure-literal >   -- 32 - bit integer constant
-    int64 < measure-literal >   -- 64 - bit integer constant
-    ieee32 < measure-literal >  -- single-precision float32 constant
-    ieee64 < measure-literal >  -- double-precision float constant
-    decimal < measure-literal > -- decimal constant
+    byte < measure-literal >                    -- 8 - bit unsigned integer constant
+    uint16 < measure-literal >                  -- 16 - bit unsigned integer constant
+    uint32 < measure-literal >                  -- 32 - bit unsigned integer constant
+    uint64 < measure-literal >                  -- 64 - bit unsigned integer constant
+    sbyte < measure-literal >                   -- 8 - bit integer constant
+    int16 < measure-literal >                   -- 16 - bit integer constant
+    int32 < measure-literal >                   -- 32 - bit integer constant
+    int64 < measure-literal >                   -- 64 - bit integer constant
+    ieee32 < measure-literal >                  -- single-precision float32 constant
+    ieee64 < measure-literal >                  -- double-precision float constant
+    decimal < measure-literal >                 -- decimal constant
```

Comment:
Should `int <...>` / `uint <...>` be mentioned? (see also comment further below)

Reply:
See previous comments / discussion about BNF vs F#